superset/db_engine_specs.py

```diff
@@ -814,6 +822,178 @@ def get_view_names(cls, inspector, schema):
         """
         return []
 
+    @classmethod
+    def _create_column_info(cls, column: RowProxy, name: str, data_type: str) -> dict:
+        """
+        Create column info object
+        :param column: column object
+        :param name: column name
+        :param data_type: column data type
+        :return: column info object
+        """
+        return {
+            'name': name,
+            'type': data_type,
+            # newer Presto no longer includes this column
+            'nullable': getattr(column, 'Null', True),
+            'default': None,
+        }
+
+    @classmethod
+    def _get_full_name(cls, names: list) -> str:
```

Comment:
Sorry for lists and dictionaries can you use the `typing` `List` and `Dict` objects respectively? This should require for you to explicitly define the types, i.e., `List[str]`.

Reply:
No problem!

Reply:
@john-bodley We merged this earlier than we should have so I opened a new PR to address feedback in a new commit... then we can merge all of it to master in one go, with feedback. :-)